requirements.txt

```diff
@@ -13,3 +13,6 @@ msgpack>=1.1.0
 
 # AWS Redshift connector
 redshift_connector==2.1.8
+
+# hip-tests
+rpm
```

Comment:
What is `rpm` needed for?

Reply:
hip-tests artifacts is packaged into deb, tar,  rpm files. 
So we need "sudo apt install rpm " on the system to generate the rpm artifact file. 

I see linux packages are manually installed in the instructions. Any way to have this rpm package installed ?
or I can skip rpm for now